### ISSUE TITLE
Issuer Not Verified in VCI Directory

### ISSUE THREAD

**Issue by Curtistic** [damaguire/SmartHealthCardScanner#2]
New Brunswick has issued their healthcard QR codes. I tried to read mine with your website, but got the error in the title. The issuer is "https://www.gnb.ca/smarthealth" Only part of the information was parsed, but it shows in the full JWS.

**Comment by damaguire** [comment]
Apologies for the troubles you're having! The not "Issuer Not Verified in VCI Directory" isn't really an error but more of a warning that the issuer of your Smart Health Card isn't registered with the VCI Directory (located here: https://github.com/the-commons-project/vci-directory/blob/main/vci-issuers.json). The VCI Directory is, at least to my knowledge, the place where all issuers are to be listed if they want to be verified. I will look into if there is a different list I can scan through that might contain New Brunswick.

Only part of the information being parsed is a harder issue to address though. Which information wasn't parsed? It could be due to a bug or it might be that New Brunswick changed something in the JWS that my code gets tripped up on. A bit more information would be greatly appreciated!

**Comment by Curtistic** [comment]
Thanks. I noticed the parsing issue also in the previous issue. The part not showing is the identity of the second shot. The first shot Moderna ("MOD") shows fine, but the second Pfizer ("PFR") is absent. Both dates show fine.

**Comment by damaguire** [comment]
Yeah the previous issue was my code not accounting for AstraZeneca at all (wasn't on the US approved list so I missed it for Canada). I will look into why yours wont parse the second Pfizer shot though. I think it's just a simple issue with the logic that I should be able to fix quickly. Will post another comment when I get it working!

**Comment by damaguire** [comment]
Just pushed a change that should resolve your parsing issue. Let me know if it doesn't or if something else pops up!

**Comment by Curtistic** [comment]
Sorry, the only thing that changed is that it now has Pfizer as my first shot instead of Moderna. Appears not to support mixed doses.
